index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css"
+    />
+    <link rel="stylesheet" href="index.css" />
+    <title>SA AUNG HTET NYEIN</title>
+  </head>
+  <body>
+    <!-- the toolbar -->
+    <nav>
+      <p class="logo">SASA</p>
```

Comment:
- [ ] Please make sure you wrap your logo in an `a` tag. 🙏  This is a good convention, semantically better, and would give a better user experience(UX)